.github/ISSUE_TEMPLATE/credentials-access.md

```diff
@@ -0,0 +1,25 @@
+--
+name: Credential Access Request
+about: Request access to the OpenTelemetry credentials store
+title: 'REQUEST: Credential Access for <USERNAME>'
+labels: area/repo-maintenance
+assignees: ''
+---
+
+<!-- Please remember to change the title of this issue by replacing
+ <USERNAME> with the actual github username -->
+
+### Repository Vault Requested
+
+e.g. https://github.com/open-telemetry/community
```

Comment:
do we have a vault for this repo? maybe pick a different example(?)

Reply:
Good point, updated the repo